swap_meet/item.py

```diff
@@ -1,2 +1,24 @@
+# ******** Wave 3 ********
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category=None, condition = 0):
+        if category is None:
```

Comment:
The category default does not have to be `None`, it can be `""`. Strings are immutable in Python. We only use `None` for defaults for mutable things like lists or dictionaries.